lib/report_portal/cucumber/report.rb

```diff
@@ -169,6 +207,10 @@ def time_to_send(desired_time)
         ReportPortal.last_used_time = time_to_send
       end
 
+      def tmp_dir
+        Pathname(ENV['TMPDIR'] ? ENV['TMPDIR'] : Dir.tmpdir)
```

Comment:
I'm curious why you need to check the value of TMPDIR environment variable.
It's checked by Ruby automatically.
```
irb(main):001:0> require 'tempfile'
=> true
irb(main):002:0> Dir.tmpdir
=> "/tmp"
irb(main):003:0> ENV['TMPDIR'] = '/home/abotalov/tmp'
=> "/home/abotalov/tmp"
irb(main):004:0> Dir.tmpdir
=> "/home/abotalov/tmp"
```
https://github.com/ruby/ruby/blame/master/lib/tmpdir.rb#L26